source/collections/Vector2D.ooc

```diff
@@ -38,6 +40,58 @@ Vector2D: class <T> {
 	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
 		result := columnCount * row + column
 	}
+	resize: func (newRowCapacity, newColumnCapacity: Int) {
+		if (newColumnCapacity == this columnCapacity)
+			if (newRowCapacity == this rowCapacity)
+				return
+			else
+				_allocate(newRowCapacity, newColumnCapacity)
```

Comment:
Again I'm confused by the names. Or maybe I just don't understand the basics - is the data layout in this class *row-major* or *column-major* ?